src/sentry/models/deletedorganization.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+from django.db import models
+
+from sentry.db.models import sane_repr
+from sentry.models.deletedentry import DeletedEntry
+
+
+class DeletedOrganization(DeletedEntry):
```

Comment:
Let's comment on this table what cases this covers and what it doesn't, just so it's context that we don't lose. (I'm referring to all 3 of them)

It'd be worth noting that, for example, DeletedProject won't exist when a parent thing is deleted. Another thing this isn't tracking and I don't have a great answer for right now, so let's just document it, is that this is tracking a deletion _intent_. Not that it's truly deleted. So maybe we can change wording on the columns as well? Instead of `date_deleted`, maybe like, `date_intent`. I think this is an important distinction to make because where we're hooking in, we don't know if the deletion truly finished. It could be reversed or aborted or failed for some other reason. (side note, it might be relatively easy to follow up later and augment this table when it was actually deleted, but we can investigate this later)

Reply:
I feel there are several ways to go about fixing the fact that there are implicit deletions. I was looking at how you delete things and when. I realize that there are Sentry signals and that there is in fact a signal called pending_deletion. However, I haven't yet been able to figure out exactly how to hook into it.

Reply:
I think ```date_intent``` might be confusing. Perhaps we're better offer referring to it as something more like ```date_pending_delete``` or ```date_marked_deleted``` or something.... eh. Naming things is hard. I only worry that ```date_intent``` won't mean anything to anyone who doesn't know how we've built the code.

Reply:
Yeah, that's fair. I agree though. Also, this is why comments become valuable and we should absolutely document that records within this table represent an intent to delete, not that the deletion truly happened.